front/app/containers/MainHeader/index.tsx

```diff
@@ -8,55 +8,31 @@ import DesktopNavbarContent from './Components/NavbarContent/DesktopNavbarConten
 import Fragment from 'components/Fragment';
 import TenantLogo from './Components/TenantLogo';
 
-// hooks
-import useLocale from 'hooks/useLocale';
-
-// utils
-import {
-  isAdminPage,
-  isIdeaPage,
-  isInitiativePage,
-  isProjectPage,
-} from './utils';
-
 // style
 import styled from 'styled-components';
 import { media, isRtl } from 'utils/styleUtils';
 import { useBreakpoint } from '@citizenlab/cl2-component-library';
 
-const Container = styled.header<{ position: 'fixed' | 'absolute' }>`
+const Container = styled.header`
   width: 100vw;
   height: ${({ theme }) => theme.menuHeight}px;
   display: flex;
   align-items: stretch;
-  position: ${(props) => props.position};
+  position: fixed;
   top: 0;
   left: 0;
   background: ${({ theme }) => theme.navbarBackgroundColor || '#fff'};
   box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.1);
   z-index: 1004;
 
-  &.hideNavbar {
-    ${media.tablet`
-      display: none;
-    `}
-  }
```

Comment:
I found this a bit unclear, then realized we also declare showFrontOfficeNavbar in App/index.tsx. So I moved this logic to that page.